gravitee-apim-repository/gravitee-apim-repository-test/src/test/resources/data/plan-tests/plans.json

```diff
@@ -8,6 +8,7 @@
       "security": "API_KEY",
       "validation": "AUTO",
       "type": "API",
+      "apiType": "native",
```

Comment:
probably not important, but it's strange to see it in lowercase when all the other constants are in uppercase 🙈

Reply:
Because the actual value is in lower case
<img width="413" alt="Screenshot 2024-11-07 at 11 52 28" src="https://github.com/user-attachments/assets/45699216-4446-4c18-9b40-11fde6cb95ec">